e2e/plotting.spec.ts

```diff
@@ -527,3 +527,34 @@ test('user can plot channels on the right y axis', async ({ page }) => {
     })
   ).toMatchSnapshot({ maxDiffPixels: 150 });
 });
+
+test('user can customize y axis labels', async ({
+  page,
+  context,
+  browserName,
+}) => {
+  await page.goto('/');
+
+  await page.locator('text=Plots').click();
+
+  // open up popup
+  const [popup] = await Promise.all([
+    page.waitForEvent('popup'),
+    page.locator('text=Create a plot').click(),
+  ]);
+
+  await popup.locator('label:has-text("Search")').fill('Shot Num');
+
+  await popup.getByRole('option', { name: 'Shot Number', exact: true }).click();
+  await popup.getByRole('textbox', { name: 'Label' }).type('left y axis');
+  await popup.getByRole('button', { name: 'Right' }).click();
+  await popup.getByRole('textbox', { name: 'Label' }).type('right y axis');
```

Comment:
This test adds a right y axis label, but never selects a channel to put on the right Y axis for it to show up. Either add a channel & update the snapshots to test the right Y axis label (probably better to test it), or remove this code.